runconfig/hostconfig.go

```diff
@@ -202,6 +232,7 @@ func (c *CapList) UnmarshalJSON(b []byte) error {
 	return nil
 }
 
+// Len returns the number of specific lxc configuration
```

Comment:
I wouldn't say `lxc` here, unless I'm mistaken this is specifically about caps available to a user in a container, not really lxc specific.